doc/changelog/07-vernac-commands-and-options/14596-master+names-non-dependent-about.rst

```diff
@@ -0,0 +1,5 @@
+- **Changed:**
+  :cmd:`About` now displays all known argument names
```

Comment:
It seems that it is not only `About` that changed, but also `Print`.  Is this deliberate?

Reply:
The change of `Print` was not conscious but since `Print` prints an `Argument` line, this has to be changed too. I'll update the change loc. (But see also #14486.)